docs/_snippets/portable-stories-vitest-compose-story.md

```diff
@@ -1,19 +1,26 @@
 ```tsx filename="Button.test.tsx" renderer="react" language="ts"
 import { vi, test, expect } from 'vitest';
-import { render, screen } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import { composeStory } from '@storybook/react';
 
 import meta, { Primary } from './Button.stories';
 
-test('onclick handler is called', () => {
+test('renders primary button with default args', async () => {
   // Returns a story which already contains all annotations from story, meta and global levels
   const PrimaryStory = composeStory(Primary, meta);
 
-  const onClickSpy = vi.fn();
-  render(<PrimaryStory onClick={onClickSpy} />);
-  const buttonElement = screen.getByRole('button');
-  buttonElement.click();
-  expect(onClickSpy).toHaveBeenCalled();
+  await PrimaryStory.play();
+  const buttonElement = screen.getByText('Text coming from args in stories file!');
+  expect(buttonElement).not.toBeNull();
+});
+
+test('renders primary button with overridden props', async () => {
+  // You can override props and they will get merged with values from the story's args
+  const PrimaryStory = composeStory({ ...Primary, args: { label: 'Hello world' } }, meta);
```

Comment:
@yannbf — Is this still the correct/preferred way to override props when using `composeStory`?

Reply:
I'd say it should be the same as with `composeStories` – you override them in the play function